### PR TITLE
Soil grids data

### PR DESCRIPTION
**Pull request by kmdeck**
Checklist:
- [X] I created a new folder `$artifact_name`
  - [X] I added a `README.md` in that that folder that
    - [X] describes the data and processing done to it
    - [X] lists the sources of the raw data
    - [X] lists the required citation, licenses
  - [x] If applicable (e.g., for Creative Commons), I added a `LICENSE` file
  - [X] I added the scripts that retrieve, process, and produce the artifact
  - [X] I added the environment used for such scripts (typically, `Project.toml`
        and `Manifest.toml`)
  - [X] I added the `OutputArtifacts.toml` file containing the information
        needed for package developers to add `$artifact_name` to their package
- [X] I uploaded the artifact folder to the Caltech cluster (in
      `/groups/esm/ClimaArtifacts/artifacts/$artifact_name`)
- [X] I added the relevant code to the `Overides.toml` on the Caltech Cluster
      (in `/groups/esm/ClimaArtifacts/artifacts/Overrides.toml`)
- [x] I added a link to the main `README.md` to point to the new artifact

### CODE REVIEW

**Review by imreddyTeja — APPROVED** [review]
Looks good. The PR checklist says to add the `Manifest.toml`, but there are some artifacts without it, so it might not be needed.

**Comment by Sbozzolo** [comment]
Yes, if you can add the Manifest.toml, it would be best. Thank you!

File: soilgrids/create_artifacts.jl [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+# Here, nvars, vars, attribs, transforms, level_names, and the pre-allocated
+# memory `data` were created in "combine_and_transform_raw_data.jl", along
+# with the function `create_combined_data`.
+for i in 1:nvars
+    var = vars[i]
+    attrib = attribs[i]
+    transform = transforms[i]
+    files = ["$(var)_$(ln)_mean_5000.nc" for ln in level_names]
+    outfilepath = "$(var)_soilgrids_combined.nc"
+    create_combined_data(data, files, attrib, transform, outfilepath)
```
[the remark this answers is not in the document]
Comment (kmdeck) [reply soilgrids/create_artifacts.jl:35]: thanks! changed in both places

File: soilgrids/create_artifacts.jl [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+# min = silt, clay, sand
+
+# - q_i = mass of i/mass of fine earth
+# - f_i = mass of i/mass of minerals in fine earth
+# - θ_i = volumetric fraction of soil component relative to whole soil
+# - q_soc + q_min = 1 (fine earth mass fractions)
+# - f_i * (1-q_soc) = q_i (fine earth mass fraction of mineral i)
+# - f_silt .+ f_sand .+ f_clay = 1; # sums to 1 in the data
+# - θ_fe + θ_cf = 1, porosity included in θ_fe
+
```
[the remark this answers is not in the document]
Comment (kmdeck) [reply soilgrids/create_artifacts.jl:55]: I added:
# - ν_fe = volume of pores in fine earth/volume of fine earth
# - ν = volume of pores/volume of whole soil
# - ν_ss_i = volume of soil solid relative to soil solids (incudes fine and coarse soil components)

[the remark this answers is not in the document]
Comment (kmdeck) [reply soilgrids/create_artifacts.jl:55]: I realized that the notation might be better if nu was just for pores, but this matches the notation of the paper

**Comment by kmdeck** [comment]
Here are the surface level maps:
sand:
sand
organic matter:
om
coarse fragments:
cf
all except minerals:
all

**Pull request merged**